hack/publish.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Copyright 2019 The Volcano Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+
+# The process of preparing volcano release.
+#   1. cp binaries into release folder
+#   2. cp README document into release folder
+#   3. cp default queue into release folder
+#   4. cp helm charts template into release folder and update default image tag
+#   5. upload docker images to volcano.sh
+#   6. generate zip file
+
+VK_ROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..
+BINARY_FOLDER=${VK_ROOT}/${BIN_DIR}/${REL_OSARCH}
+RELEASE_FOLDER=${VK_ROOT}/${RELEASE_DIR}
+RELEASE_BINARY=${RELEASE_FOLDER}/bin
+QUEUE_FILE=${VK_ROOT}/installer/helm/chart/volcano/templates/default-queue.yaml
+README_FILE=${VK_ROOT}/installer/README.md
+HELM_FOLDER=${VK_ROOT}/installer/helm
+VOLCANO_IMAGE_TAG=${TAG:-"latest"}
+DOCKER_PASSWORD=${DOCKER_PASSWORD:-""}
```

Comment:
we need to first add these environment variables in travis CI settings

Reply:
Sure, it's mentioned above. I don't have the rights.

Reply:
okay I have added it now
![image](https://user-images.githubusercontent.com/9736066/61289357-c7540600-a7e6-11e9-8193-a909796ad856.png)